yt_dlp/extractor/disney.py

```diff
@@ -101,6 +125,10 @@ def _real_extract(self, url):
         for flavor in video_data.get('flavors', []):
             flavor_format = flavor.get('format')
             flavor_url = flavor.get('url')
+            if '/emea-exit/' in flavor_url:
+                webpage = self._download_webpage(flavor_url, display_id, headers={'user-agent': self._USER_AGENT}, note=False)
```

Comment:
why `note=False`? typically we always want to communicate when a web request is being made

Reply:
Idk, it just looked messy with a bunch of `Downloading webpage` messages in a row and I didn't think it was important to output the redirects to the console. But I've changed it to add a more descriptive note instead now.